spec/features/user_sees_activities_page_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+describe 'User features' do
+  it 'renders the index page' do
+    visit activities_path
+
+    expect(page).to have_content("Plan Trip by Activities")
+  end
+
+  it 'has more information links' do
+    visit activities_path
+
+    expect(page).to have_content("More Information")
+  end
+
+  it 'has a login link' do
+    visit activities_path
+
+    expect(page).to have_content("Login")
+  end
+
+  it 'has a register link' do
+    visit activities_path
+
+    expect(page).to have_content("Register")
+  end
+
+  it 'has a cart link that works' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.